src/styles/_calendar.scss

```
@@ -0,0 +1,22 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: $calendar-width;
+
+  &__day {
+    @include cell($cell-size);
+    border: 1px solid #000;
+    background-color: $cell_color;
+    transition: background-color 0.5s ease-in-out, transform 0.5s ease-in-out;
```

Create variable for transitionDuration
```suggestion
    transition: background-color $transitionDuration ease-in-out, transform $transitionDuration ease-in-out;
```